registry/storage/garbagecollect_test.go

```diff
@@ -215,11 +186,21 @@ func TestDeleteManifestIfTagNotFound(t *testing.T) {
 		t.Fatalf("failed to make layers: %v", err)
 	}
 
+	digests1 := []digest.Digest{}
```

Comment:
the original coode was calling `getKeys(randomLayers1)` and `getKeys(randomLayers2)` when passing the params to `estutil.MakeSchema1Manifest`; we could have kept that helper maybe; this code just makes it explicit because `getKeys` was removed (I'm assuming as part of the removal of `testutil.MakeSchema1Manifest` calls, and then never put back leading to this code. But this code is needed to pass the digests down to `testutil.MakeSchema2Manifest` instead of the schema1 `estutil.MakeSchema1Manifest` call. Maybe a bit of an oversight but not incorrect. @DavidSpek do you wanna put back `getKeys` and then pass `getKeys(randomlayers[12])` down to schema2 manifet func

Reply:
That comment was more for my understanding than a request for change, happy to go with either style.

Reply:
I think we should put that func (getKeys) back either way. This is a great catch even if we arrived at it by tangent 🙃

Reply:
Added back the `getKeys()` function and am passing that to `testutil.MakeSchema2Manifest` now.